apps/web/src/pages/templates/TemplatesListNoData.tsx

```diff
@@ -11,66 +16,158 @@ const NoDataHolder = styled.div`
   height: 500px;
 `;
 
-const NoDataHeading = styled.h2`
-  font-size: 28px;
-  font-weight: bold;
-  color: ${colors.B40};
-`;
-
 const NoDataSubHeading = styled.p`
   margin: 0;
   font-size: 20px;
-  color: ${colors.B40};
+  color: ${({ theme }) => (theme.colorScheme === 'dark' ? colors.B40 : colors.B80)};
+`;
+
+const Card = styled.button`
+  outline: none;
+  border: none;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 16px;
+  min-width: 140px;
+  width: 140px;
+  height: 100px;
+  border-radius: 8px;
+  color: ${colors.B60};
+  background: ${({ theme }) => (theme.colorScheme === 'dark' ? colors.B20 : colors.B98)};
+  box-shadow: ${shadows.dark};
+  font-size: 14px;
+  transition: all 0.25s ease;
+
+  > svg {
+    font-size: 20px;
+  }
+
+  &:disabled {
+    cursor: not-allowed;
+  }
+
+  &:not(:disabled)&:hover {
+    color: ${({ theme }) => (theme.colorScheme === 'dark' ? colors.white : colors.B60)};
+    background: ${({ theme }) => (theme.colorScheme === 'dark' ? colors.B30 : colors.BGLight)};
+  }
+
+  &[data-can-be-hidden='true'] {
+    &:nth-last-of-type(2) {
+      display: none;
+    }
+
+    @media screen and (min-width: 1369px) {
+      &:nth-last-of-type(2) {
+        display: flex;
+      }
+    }
+  }
+
+  @media screen and (min-width: 1025px) {
+    min-width: 160px;
+    width: 160px;
+    height: 120px;
+
+    > svg {
+      font-size: 24px;
+    }
+  }
 `;
 
 const CardsContainer = styled.div`
   display: flex;
-  gap: 40px;
-  margin: 50px 40px;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 20px;
+  margin: 50px 20px;
+
+  @media screen and (min-width: 1025px) {
+    gap: 24px;
+    margin: 50px 40px;
+  }
+`;
+
+const SkeletonIcon = styled(Skeleton)`
+  min-width: 24px;
+  width: 24px;
+  height: 24px;
+
+  @media screen and (min-width: 1025px) {
+    min-width: 28px;
+    width: 28px;
+    height: 28px;
+  }
 `;
 
 export const TemplatesListNoData = ({
```

Comment:
when there is no data on the templates list page this component is shown instead